wagtail/admin/templates/wagtailadmin/tables/table.html

```diff
@@ -1,6 +1,11 @@
 {% load wagtailadmin_tags %}
 
 <table{% include "wagtailadmin/shared/attrs.html" with attrs=table.attrs %}>
+    {% with caption=table.get_caption %}
+        {% if caption %}
+            <caption class="w-sr-only">{{caption}}</caption>
```

Comment:
```suggestion
            <caption class="w-sr-only">{{ caption }}</caption>
```

Only nitpick is just to remember to space out variable usage here, it's not enforced with the linting but most of our template variables are written like this. It helps for readability in the HTML.